README.md

```diff
@@ -1016,10 +1285,17 @@ const closeConnection = conn => Future((rej, res) => {
 ```
 
 #### finally
-##### `#finally :: Future a b ~> Future a c -> Future a b`
-##### `#lastly :: Future a b ~> Future a c -> Future a b`
-##### `.finally :: Future a c -> Future a b -> Future a b`
-##### `.lastly :: Future a c -> Future a b -> Future a b`
+
+<details><summary><code>finally :: Future a b ~> Future a c -> Future a b</code></summary>
+
+```hs
+finally :: Future a b ~> Future a c               -> Future a b
+lastly  :: Future a b ~> Future a c               -> Future a b
+finally ::               Future a c -> Future a b -> Future a b
+lastly  ::               Future a c -> Future a b -> Future a b
```

Comment:
`Future.finally` would be misleading in the context of the ES6 module, as the named import `Future` has no property `finally`. Instead, `finally` is a named import that lives alongside `Future`.